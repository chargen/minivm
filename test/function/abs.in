print abs(10)
print abs(-10)
print abs(0)

print abs(3.14)
print abs(-6.12)
