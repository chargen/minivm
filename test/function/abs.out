10
10
0
3.140000000
6.120000000
